src/fragments/lib/project-setup/flutter/prereq/prereq.mdx

```diff
@@ -1,20 +1,2 @@
 - Install [Flutter](https://flutter.dev/docs/get-started/install) version 2.0.0 or higher (make sure you are using a stable version of flutter)
 - Setup your [IDE](https://flutter.dev/docs/get-started/editor?tab=androidstudio)
-- Android API level 21 (Android 5.0) or above
-- iOS platform version of at least 11.0.
-
-<Callout warning>
```

Comment:
@salihgueler - do we no longer need this because of Flutter 3? I remember we ran into this issue when developers had to upgrade their apps from versions below 2.10

Reply:
I removed these because we already set it up in the getting started part of the libraries. It felt repetetive. Also these did not show how the increase these values, the guide shows them.